xgi/drawing/xgi_pylab.py

```diff
@@ -193,12 +191,8 @@ def draw_xgi_nodes(
         * max_node_size
     """
     # Note Iterable covers lists, tuples, ranges, generators, np.ndarrays, etc
-    node_fc = _color_arg_to_dict(
-        node_fc, H.nodes, settings["node_fc_cmap"]
-    )
-    node_ec = _color_arg_to_dict(
-        node_ec, H.nodes, settings["node_lc_cmap"]
-    )
+    node_fc = _color_arg_to_dict(node_fc, H.nodes, settings["node_fc_cmap"])
+    node_ec = _color_arg_to_dict(node_ec, H.nodes, settings["node_lc_cmap"])
```

Comment:
Did you change black settings? Or were these lines previously not formatted with black? We should make sure we all have the same black settings.

Reply:
I updated the black version in the developer requirements a few months ago, so maybe that's it?